bin/blocks-continue

```diff
@@ -1,15 +0,0 @@
-#!/usr/bin/env python
-import logging
-from argparse import ArgumentParser
-
-from blocks.scripts import continue_training
-
-if __name__ == "__main__":
-    logging.basicConfig()
-
-    parser = ArgumentParser("Continues your pickled main loop")
-    parser.add_argument(
-        "path", help="A path to a file with a pickled main loop")
-    args = parser.parse_args()
-
-    continue_training(args.path)
```

Comment:
Right, this is true, but I still like `blocks-continue`. I just got a habit of having only parameter parsing in the runnable `main.py` script.